### PR TITLE
Feature/additional tests

### PR DESCRIPTION
**Pull request by rewhowe**
Added tests for Conjugator, Scope, and Reader. Also fixed a few random edge cases here and there.

### CODE REVIEW

File: spec/tokenizer/conjugator_spec.rb [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+
+  describe '#conjugate' do
+    it 'conjugates 五段 verbs ending in う' do
+      expect(Conjugator.conjugate('買う')).to contain_exactly('買って', '買った')
+    end
+
+    it 'conjugates 五段 verbs ending in く' do
+      expect(Conjugator.conjugate('開く')).to contain_exactly('開いて', '開いた')
+    end
+
```
Comment (rewhowe) [inline spec/tokenizer/conjugator_spec.rb:72]: Missing test for ぐ.
